sdk/resourcemanager/azure-resourcemanager-samples/src/main/java/com/azure/resourcemanager/appservice/samples/ManageWebAppSourceControlAsync.java

```diff
@@ -70,9 +70,9 @@ public static boolean runSample(final Azure azure) {
                     .withJavaVersion(JavaVersion.JAVA_8_NEWEST)
                     .withWebContainer(WebContainer.TOMCAT_8_0_NEWEST)
                     .createAsync()
-                    .flatMap(indexable -> {
+                    .flatMapMany(indexable -> {
                         if (indexable instanceof WebApp) {
```

Comment:
Likely this `if` is no longer needed?

Reply:
Yes, there are lots of `if` not needed, but I don't want to change so much.

Reply:
fixed, it cause spotbugs error